schemas/oic.r.coapcloudconf-schema.json

```diff
@@ -1,43 +1,38 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
-  "id": "https://www.openconnectivity.org/ocf-apis/core/schemas/oic.r.coapcloudconf-schema.json#",
+  "id": "https://www.openconnectivity.org/ocf-apis/core-extensions/schemas/oic.r.coapcloudconf-schema.json#",
   "definitions": {
     "oic.r.coapcloudconf": {
       "type": "object",
       "properties": {
         "apn": {
           "type": "string",
-          "description": "Authentication provider's name issuing the auth code",
+          "description": "The Authorisation Provider through which an Access Token was obtained.",
           "pattern": "^.*$"
```

Comment:
What is this pattern trying to define?

Reply:
It means all characters are valid in string.

Reply:
None of our other "string" Properties have this pattern.  If it can be anything, why even have this.  You can remove the "pattern" and it means the exact same thing.  There is no value in having this and it only leads to questions.  If you don't have a clear need for this, can we remove it?  Especially since we don't do it anywhere else.

Reply:
As discussed over email, removed generic pattern. Updated in PR #5 (Now Merged).